### ISSUE TITLE
issues and `non-recoverable error` in Factorio 2.0 Space Age

### ISSUE THREAD

**Issue by SoongJr** [321freddy/even-distribution#49]
Just installed Space Age, updated mods, and re-activated them. Started a new game from scratch and had some issues:
- Ctrl+Drag does not work, only considers the first one or two entity for distribution (looks like the timer doesn't work?)
- Shit+C works intermittently, sometimes it does nothing. I thought maybe I'm just not understanding the thresholds correctly, but then I got the big one:
- after an hour or so of gameplay, tried again to distribute with Ctrl+Drag on some stone furnaces (just out of habit, not expecting it to work) and got a crash to main menu (had to reload a save). Here's the message:
  ```
  3324.245 Error MainLoop.cpp:1429: Exception at tick 178415: The mod Even Distribution (2.0.0) caused a non-recoverable error.
  Please report this error to the mod author.
  
  Error while running event even-distribution::on_player_fast_transferred (ID 140)
  LuaGameScript doesn't contain key get_filtered_recipe_prototypes.
  stack traceback:
	  [C]: in function '__index'
	  __even-distribution__/scripts/helpers/LuaEntity.lua:32: in function 'canSmelt'
	  __even-distribution__/scripts/drag.lua:319: in function 'isEntityEligible'
	  __even-distribution__/scripts/drag.lua:225: in function 'onStackTransferred'
	  __even-distribution__/scripts/drag.lua:210: in function 'func'
	  __even-distribution__/framework.lua:49: in function <__even-distribution__/framework.lua:48>
  ```

Not many other mods installed, just a subset of what I had in Factorio 1 (without those that don't have updates yet). Here is the full log file:
[factorio-EvenDistCrash.log](https://github.com/user-attachments/files/17465361/factorio-EvenDistCrash.log)

Let me know if you need any more info.

**Comment by 321freddy** [comment]
These have already been fixed in 2.0.1, please update the mod ;)

**Comment by SoongJr** [comment]
> These have already been fixed in 2.0.1, please update the mod ;)

25 minutes before I opened the issue, meaning I was probably still compiling it when the update came out 🤣


Thanks! Not only for the fix, but also the mod in general.
